Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
+    "setup_per_page_switcher" (subject) {
+
+        Admin.log("[core|setup_per_page_switcher] setup page switcher", subject);
+
+        jQuery("select.per-page").change(function (event) {
 
-        jQuery('select.per-page').change(function(event) {
-            jQuery('input[type=submit]').hide();
+            jQuery("input[type=submit]").hide();
+
+            window.top.location.href = this.options[this.selectedIndex].value;
 
-            window.top.location.href=this.options[this.selectedIndex].value;
         });
+
     },
 
-    setup_form_tabs_for_errors: function(subject) {
-        Admin.log('[core|setup_form_tabs_for_errors] setup form tab\'s errors', subject);
+    "setup_form_tabs_for_errors" (subject) {
+
+        Admin.log("[core|setup_form_tabs_for_errors] setup form tab's errors", subject);
 
         // Switch to first tab with server side validation errors on page load
-        jQuery('form', subject).each(function() {
-            Admin.show_form_first_tab_with_errors(jQuery(this), '.sonata-ba-field-error');
+        jQuery("form", subject).each(function () {
+
+            Admin.show_form_first_tab_with_errors(jQuery(this), ".sonata-ba-field-error");
+
         });
 
         // Switch to first tab with HTML5 errors on form submit
-        jQuery(subject)
-            .on('click', 'form [type="submit"]', function() {
-                Admin.show_form_first_tab_with_errors(jQuery(this).closest('form'), ':invalid');
-            })
-            .on('keypress', 'form [type="text"]', function(e) {
-                if (13 === e.which) {
-                    Admin.show_form_first_tab_with_errors(jQuery(this), ':invalid');
+        jQuery(subject).
+            on("click", "form [type=\"submit\"]", function () {
+
+                Admin.show_form_first_tab_with_errors(jQuery(this).closest("form"), ":invalid");
+
+            }).
+            on("keypress", "form [type=\"text\"]", function (e) {
+
+                if (e.which === 13) {
+
+                    Admin.show_form_first_tab_with_errors(jQuery(this), ":invalid");
+
                 }
-            })
-        ;
+
+            });
+
     },
 
-    show_form_first_tab_with_errors: function(form, errorSelector) {
-        Admin.log('[core|show_form_first_tab_with_errors] show first tab with errors', form);
+    "show_form_first_tab_with_errors" (form, errorSelector) {
+
+        Admin.log("[core|show_form_first_tab_with_errors] show first tab with errors", form);
 
-        var tabs = form.find('.nav-tabs a'), firstTabWithErrors;
+        let tabs = form.find(".nav-tabs a"),
+            firstTabWithErrors;
 
-        tabs.each(function() {
-            var id = jQuery(this).attr('href'),
+        tabs.each(function () {
+
+            let id = jQuery(this).attr("href"),
                 tab = jQuery(this),
-                icon = tab.find('.has-errors');
+                icon = tab.find(".has-errors");
```

Comment:
- eslint: Variables within the same declaration block should be sorted alphabetically.
- eslint: 'icon' is never reassigned. Use 'const' instead.